tests/languages/dart/tests.dart

```diff
@@ -114,7 +114,7 @@ void main() async {
   print(Permission.update(Role.user(ID.custom('userid'), 'unverified')));
 
   // ID helper tests
-  print(ID.unique());
+  // print(ID.unique());
```

Comment:
we can remove this or we can test this by printing twice making sure it gives back unique ID each time?

Reply:
great idea!